kfs-core/src/main/resources/edu/arizona/kfs/pdp/spring-pdp.xml

```diff
@@ -5,6 +5,22 @@
                            http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
 
     <bean id="pdpModuleConfiguration" parent="pdpModuleConfiguration-parentBean" >
+    	<property name="namespaceCode" value="KFS-PDP"/>

```

Comment:
This property should be inherited from the parent bean in the baseline.  Please remove as it shouldn't be needed.